sku.routes.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+const partition = require('lodash/partition');
+
+const componentsPath = path.join(__dirname, 'lib/components/index.js');
+const componentsSource = fs.readFileSync(componentsPath, 'utf-8'); // eslint-disable-line no-sync
+const components = componentsSource
+  .match(/default as [A-Za-z]+/g)
+  .map(x => x.replace('default as ', ''))
+  .sort();
+const [iconNames, componentNames] = partition(components, x =>
+  /.+Icon$/i.test(x)
```

Comment:
The `.+` is kinda redundant here

Reply:
It's enforcing that Icon is a suffix, mostly to guard against `Icon` being included in the "Icons" category if we ever export it.

Reply:
The `$` is enforcing it is a suffix. But if you dont want to match on `Icon` thats ok i guess

Reply:
Not quite, `/Icon$/` will still match `Icon`.